### DISCUSSION TITLE
Bulk selecting/adding a CTA to tables

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#2299] [Office hours] [answered]
Hi! I was wondering if there's a way to apply checkboxes to the current table component in order to add bulk action functionality: 

![Screen Shot 2022-03-16 at 3 49 02 PM](https://user-images.githubusercontent.com/87141551/158696868-a3e9c179-9a49-4bbb-b9c6-ffdc8bd7c408.png)

Also wondering is there's a best practice for adding a CTA to a table (if it should be contained in the footer, etc.

![Screen Shot 2022-03-16 at 3 51 46 PM](https://user-images.githubusercontent.com/87141551/158697231-30c4a08e-f342-4590-95ef-108a3e9e82e2.png)

**Comment by nkrantz** [comment]
Hi @ceyerkes! I see you signed up for Office Hours on tues, just wanted to let you know that the team will review this post (and the other one) beforehand so that we're ready to dive in.

**Reply by ceyerkes** [reply]
Sounds good!

**Comment by shleewhite** [comment, marked as answer]
Following up from office hours today, we talked about:
* Placing the "Share business profile" button in by the filters, similar to the "export to CSV" button in [the filter group pattern](https://paste.twilio.design/patterns/filter-group#general)
* Add an extra column for the checkboxes so the text is all aligned in the same place, similar to [the data grid with selectable row example](https://paste.twilio.design/components/data-grid#data-grid-with-selectable-rows)
  * In figma, you will need to use the table with icon in the header and then change the icon to a checkbox component. You will also probably need to detach the table to adjust the width.

**Reply by ceyerkes** [reply]
Thanks Lee!
